app/models/form_profiles/va_686c674v2.rb

```diff
@@ -36,23 +36,23 @@ def metadata
   private
 
   def prefill_form_address
-    mailing_address = begin
-      VAProfileRedis::ContactInformation.for_user(user).mailing_address if user.vet360_id.present?
+    begin
+      mailing_address = if user.icn.present? && Flipper.enabled?(:remove_pciu, user)
+                          VAProfileRedis::V2::ContactInformation.for_user(user)
+                        elsif user.vet360_id.present?
+                          VAProfileRedis::ContactInformation.for_user(user)
+                        end
```

Comment:
mailing address will be present if the`user.icn.present? && Flipper.enabled?(:remove_pciu, user)` OR `user.vet360_id.present?`